crate/utils/src/crypto/cover_crypt/kmip_requests.rs

```diff
@@ -95,8 +95,9 @@ pub fn build_import_decryption_private_key_request<T: IntoIterator<Item = impl A
     //  - to wrapped (wrapping_password is some)
     //  - or not wrapped (otherwise)
     let is_wrapped = is_wrapped || wrapping_password.is_some();
+    // TODO - is salt necessary here ?
```

Comment:
since wrapping_password could be weak, salt should be necessary here. Probably create an issue with low priority.

Reply:
If I understand correctly this part of the code, the database master key is created upon database initialization and there should be no use of needing to compute it again ? If so, then there is no point in storing the salt. What do you think ?